cluster/local/minikube.sh

```diff
@@ -91,14 +93,20 @@ case "${1:-}" in
     fi
     ;;
   helm-install)
-    echo " copying image for helm"
+    echo "copying image for helm"
     helm_tag="$(cat _output/version)"
     copy_image_to_cluster ${BUILD_IMAGE} "${DOCKER_REGISTRY}/${PROJECT_NAME}:${helm_tag}"
 
     [ "$2" ] && ns=$2 || ns="${DEFAULT_NAMESPACE}"
     echo "installing helm package(s) into \"$ns\" namespace"
     helm install --name ${PROJECT_NAME} --namespace ${ns} ${projectdir}/cluster/charts/${PROJECT_NAME} --set image.pullPolicy=Never,imagePullSecrets=''
     ;;
+  helm-upgrade)
```

Comment:
this will be useful functionality to have when iterating in minikube, thanks for adding it. Consider adding it to the flow in https://github.com/crossplaneio/crossplane/tree/master/cluster/local#run-tests at some time in the future.